### PR TITLE
Id 31 map implementation

### PR DESCRIPTION
**Pull request by wojtek717**

### CODE REVIEW

**Comment by wojtek717** [comment]
[Map implementation](https://trello.com/c/mlTBSfGe/31-map-implementation)

File: kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class MapViewController: UIViewController {
+
+  @IBOutlet private weak var mapView: MKMapView!
+
+  lazy var viewModel = MapViewModel(presenter: self)
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    viewModel.centerMap(atLatitude: 51.111510, atLongitude: 17.059737, forRegionRadious: 1000)
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift]: lat/long values should be provided by View Model. In the future, we will use a device GPS to select center point

File: kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  init(presenter: MapViewControllerPresenter?) {
+    self.presenter = presenter
+  }
+
+  func centerMap(
+    atLatitude lat: Double,
+    atLongitude long: Double,
+    forRegionRadious radious: Double) {
+    let location = CLLocation(latitude: lat, longitude: long)
+    let coordinateRegion = MKCoordinateRegion(
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift]: I'm not a fan of moving the first parameter to a new line. I know this one is described in Styleguide, but we use
```
let coordinateRegion = MKCoordinateRegion(center: location.coordinate,
       latitudinalMeters: radious,
       longitudinalMeters: radious)
```

File: kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      center: location.coordinate,
+      latitudinalMeters: radious,
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift]: radius not radious

File: kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift (hunk 2 above)
Comment (pmieszal) [reply kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift]: Hey man, but in this case I think that this is good approach. When we have long name of class, breaking line before 1st parameter is increasing readability imho. 

What would you do in this case:
```
let coordinateRegion = MKCoordinateRegion(center: location.coordinate,
                                          coordinate: CLLocationCoordinate2D(longitude: longitude,
                                                                             latitude: latidude)
)
```

Comment (pmieszal) [reply kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift]: I think, this is better in every way 
```
let coordinateRegion = MKCoordinateRegion(
    center: location.coordinate,
    coordinate: CLLocationCoordinate2D(
        longitude: longitude,
        latitude: latidude)
)
```

Comment (leszko11) [reply kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift]: I don't like to move first param regardless of line length shrug ¯\_(ツ)_/¯
@wojtek717 choose one 😅

**Pull request merged**